libminifi/include/FlowController.h

```diff
@@ -197,6 +202,8 @@ class FlowController {
 	std::atomic<bool> _initialized;
 	//! Provenance Repo
 	ProvenanceRepository *_provenanceRepo;
+	//! FlowFile Repo
+	FlowFileRepository *_flowfileRepo;
```

Comment:
Why does this need to be a pointer? Why not use unique/shared pointer depending on the need?